Running in wrapper mode for '206'

Running python job.

[name: "CPU-QUOTA"
int_val: 87
int_val: 21
int_val: 58
int_val: 100
int_val: 14
int_val: 6
int_val: 56
, name: "DISK"
int_val: 24
int_val: 29
int_val: 25
int_val: 16
int_val: 17
int_val: 14
int_val: 34
, name: "MEMORY"
int_val: 8
int_val: 10
int_val: 36
int_val: 31
int_val: 39
int_val: 16
int_val: 30
, name: "NET"
int_val: 32
int_val: 11
int_val: 13
int_val: 33
int_val: 34
int_val: 36
int_val: 25
]
The paramters are {u'CPU-QUOTA': array([ 87,  21,  58, 100,  14,   6,  56]), u'NET': array([32, 11, 13, 33, 34, 36, 25]), u'DISK': array([24, 29, 25, 16, 17, 14, 34]), u'MEMORY': array([ 8, 10, 36, 31, 39, 16, 30])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 52500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 56320.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 24326963.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 375000000.0
Recovering the memory utilization
Setting cpu quota to 217500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 163840.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 20132659.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 300000000.0
Recovering the memory utilization
Setting cpu quota to 145000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 66560.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 20971520.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 1350000000.0
Recovering the memory utilization
Setting cpu quota to 217500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 163840.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 20132659.2
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 300000000.0
Recovering the memory utilization
Setting cpu quota to 145000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 66560.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 20971520.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 1350000000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f4038478dd0>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.58 seconds.
